includes/formFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -579,6 +580,11 @@ private static function buildDescriptorPermissions(
 				'default' => wfMessage( 'managewiki-permissions-unassigned' )->text(),
 				'section' => 'unassigned'
 			],
+			'revoked' => [
+				'type' => 'info',
+				'default' => wfMessage( 'managewiki-permissions-revoked' )->text(),
```

Comment:
Does this interface message exist?

Reply:
Oh crap. I forgot to add that. Thanks. I'll do so now. I only know English so I'll only be able to add it to the English language though.